unlock-app/src/constants.ts

```diff
@@ -18,7 +18,7 @@ export const ETHEREUM_NETWORKS_NAMES = {
  * @param title
  * @returns {string}
  */
-export const pageTitle = title => {
+export function pageTitle(title: string): string {
```

Comment:
An example of migrating a function to have type annotations. Before this change, the tests complained about the implicit `any` type.

It is not an error to manually specify the `any` type, but you really shouldn't.

Reply:
Going to comment here, since GitHub doesn't let you comment on files that only had their extensions changed.

For the `CreatorLock` component, migrating it was as simple as changing the file extension to `.tsx`. Then you just have to suppress any errors--ideally by annotating the correct types for each declaration. Most of our files are not enormous, so doing this incrementally should not be a great hardship.

Reply:
> It is not an error to manually specify the any type, but you really shouldn't.

Is there an eslint (not sure how that would work because eslint run on transpiled files?) to enforce that we do not use `any`

Reply:
TSLint has a `no-any` rule. I haven't brought TSLint in yet, and I think we'll have to have a more involved discussion around the rules we'd want to enable, but this particular case is definitely something we can handle.